docs/source/Coding/Setting-up-PyCharm.md

```diff
@@ -1,57 +1,78 @@
+First, download and install the IDE edition of your choosing.
+The professional edition has integrated support for Django which can
+make certain steps easier but the community edition has everything you need.
 
-First, install Evennia on your local machine with [[Getting Started]]. If you're new to PyCharm, loading your project is as easy as selecting the `Open` option when PyCharm starts, and browsing to your game folder (the one created with `evennia --init`). We refer to it as `mygame` here. 
+After setting it up, you want to create a new project.
+1. Click on the new project button.
+2. Select the location for your project.
+You should create two new folders, one for the root of your project and one
+for the evennia game directly. It should look like `/location/projectfolder/gamefolder`
+3. Select the `Custom environment` interpreter type, using `Generate New` of type `Virtual env` using a
+compatible base python version as recommended in https://www.evennia.com/docs/latest/Setup/Installation.html#requirements
+Then choose a folder for your virtual environment as a sub folder of your project folder.
 
-If you want to be able to examine evennia's core code or the scripts inside your virtualenv, you'll
-need to add them to your project too:
+![Example new project configuration](https://i.imgur.com/kewCJwY.png)
 
-1. Go to `File > Open...`
-1. Select the folder (i.e. the `evennia` root)
-1. Select "Open in current window" and "Add to currently opened projects"
+Click on the create button and it will take you inside your new project with a bare bones virtual environment.
+To install Evennia, you can then either clone evennia in your project folder or install it via pip.
+The simplest way is to use pip.
 
-It's a good idea to set up the interpreter this before attempting anything further. The rest of this page assumes your project is already configured in PyCharm.
+Click on the `terminal` button
 
-1. Go to `File > Settings... > Project: \<mygame\> > Project Interpreter`
-1. Click the Gear symbol `> Add local`
-1. Navigate to your `evenv/scripts directory`, and select Python.exe
+![Terminal Button](https://i.imgur.com/fDr4nhv.png)
 
-Enjoy seeing all your imports checked properly, setting breakpoints, and live variable watching!
+1. Type in `pip install evennia`
+2. In this pycharm terminal window, continue the rest of the steps outlined here https://www.evennia.com/docs/latest/Setup/Installation.html#initialize-a-new-game
 
 ## Debug Evennia from inside PyCharm
 
-1. Launch Evennia in your preferred way (usually from a console/terminal)
-1. Open your project in PyCharm
-1. In the PyCharm menu, select `Run > Attach to Local Process...`
-1. From the list, pick the `twistd` process with the `server.py` parameter (Example: `twistd.exe --nodaemon --logfile=\<mygame\>\server\logs\server.log --python=\<evennia repo\>\evennia\server\server.py`)
+### Attaching to the process
+1. Launch Evennia in the pycharm terminal
```

Comment:
This will not be possible from these instructions since you never created the game dir properly.

Reply:
Right, I assumed people would follow the steps at the link above before continuing :thinking: